.github/workflows/build-and-test-orchard-core.yml

```diff
@@ -56,6 +56,20 @@ on:
           (https://github.community/t/how-to-clear-cache-in-github-actions/129038/5) until the associated issue
           (https://github.com/actions/cache/issues/2) is resolved and the feature is available not just via the RESTful
           API.
+      set-up-sql-server:
```

Comment:
I'd rather prefix these with "setup" instead of "set-up" to match the spelling of the action they're calling (but I also find it a bit weird how the usage of this verb changed, so you can't really separate the verb from the noun anymore).

Reply:
Let's be strong and **not** surrender to broken grammar! 🤺

Go **set up** that SQL Server!

Reply:
I usually think about this kind of stuff the same way as you do, but language is fluid, so what may start out as being broken might become the norm over time.

Reply:
You're right. But I think in this case, it's just an oversight, and I wouldn't carry it over to our own code base.

Reply:
The actions are only called "setup" not "set-up" to match the official "setup" actions of GitHub (e.g. `setup-dotnet`, `setup-node`). I'd keep these parameters though.